lib/path.js

```diff
@@ -408,7 +408,13 @@ var posix = {};
 
 
 function posixSplitPath(filename) {
-  return splitPathRe.exec(filename).slice(1);
+  const matches = splitPathRe.exec(filename);
+  var len = matches.length - 1;
```

Comment:
That would have been the case before too, though. I'm not sure if it's possible to get no match with the regexp that's currently being used ...